mountpoint-s3/tests/fuse_tests/fork_test.rs

```diff
@@ -24,6 +24,7 @@ fn run_in_background() -> Result<(), Box<dyn std::error::Error>> {
         .arg(mount_point.path())
         .arg(format!("--prefix={prefix}"))
         .arg("--auto-unmount")
+        .arg(format!("--region={region}"))
```

Comment:
I'm guessing this is because the tests fail if region isn't specified outside of `aws` partition?

Not sure how I feel about this, still want to keep a version of this test as is.

Reply:
I would prefer to keep the test as is for `aws` partition too. Maybe we need another environment variable to tell whether it's in `aws` partition, then we will pass in `--region` argument if it's not. Something like
```
if !commercial_partition {
        cmd.arg(format!("--region={region}"));
}
```

Reply:
Discussed a little offline.

It might be just a test that we recommend skipping outside of the commercial partition, or that is ignored by default and we can opt into.

Ultimately, I want to keep this test as "region cannot otherwise be determined, does it work by probing `us-east-1`?".

Thinking to have a test specifically for this case, and add region elsewhere like in this change.

Reply:
@ahmarsuhail can you add a copy of this test that does not include the region, and keep the `run_in_background` test including the region.

Basically want to make it very clear what the test is testing - i.e. that automatic region resolution works as expected when not provided as a CLI flag.